virttest/env_process.py

```diff
@@ -1044,6 +1044,14 @@ def preprocess(test, params, env):
         iscsidev = qemu_storage.Iscsidev(params, base_dir, "iscsi")
         params["image_name"] = iscsidev.setup()
         params["image_raw_device"] = "yes"
+        for image_name in params.objects("images"):
+            lun_tag = "lun_%s" % image_name
+            if params.get(lun_tag):
+                image_params = params.object_params(image_name)
+                lun = image_params.get("lun", "0")
```

Comment:
image_params['lun'], we should always get the value of 'lun' here

Reply:
We use lun 0 for default, so we don't define lun_image1 = 0 in our cfgs, and for some needs, we set image_name_image1 = ${image_ name} in https://gitlab.cee.redhat.com/kvm-qe/kar/-/merge_requests/924, so lun = image_params.get("lun", "0") is better here.

Reply:
I mean the if statement(1049) has already confirmed 'lun' is defined, so we can always get image_params['lun'] here, the default value '0' will never be used.

but what you mentioned here is right, we need to consider lun_image1, way may also have to give it a default value, just like what you've done for remote_nfs

Reply:
oh, I see. I don't define lun_image1=0 and in patch:924 I set image_name_image1. So I need to revert the code to the original, use "if params.get(name_tag):" instead of  "if params.get(lun_tag):" here, or else, the setting for image_name_image1 will be missed, thanks for reminding.

Reply:
what about adding lun_image1 = 0 in 924, for it meaningless to assign an image name to a block device image

Reply:
consider someone is now using hard conf:
image_name_data = /dev/sdc
Then the change now can break above.

If we use lun, which is not used for iscsi backend currently, then we don't break the settings above.